# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libtool/notinst_path-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libtool-1.5.22/ltmain.in.orig	2006-01-25 22:49:48.000000000 +0100
+++ libtool-1.5.22/ltmain.in	2006-01-25 22:50:49.000000000 +0100
@@ -3411,9 +3411,9 @@
 
       # Eliminate all temporary directories.
       for path in $notinst_path; do
-	lib_search_path=`$echo "$lib_search_path " | ${SED} -e "s% $path % %g"`
-	deplibs=`$echo "$deplibs " | ${SED} -e "s% -L$path % %g"`
-	dependency_libs=`$echo "$dependency_libs " | ${SED} -e "s% -L$path % %g"`
+	lib_search_path=`$echo " $lib_search_path " | ${SED} -e "s% $path % %g"`
+	deplibs=`$echo " $deplibs " | ${SED} -e "s% -L$path % %g"`
+	dependency_libs=`$echo " $dependency_libs " | ${SED} -e "s% -L$path % %g"`
       done
 
       if test -n "$xrpath"; then
--- libtool-1.5.22/ltmain.sh.orig	2006-01-25 22:49:40.000000000 +0100
+++ libtool-1.5.22/ltmain.sh	2006-01-25 22:50:30.000000000 +0100
@@ -3411,9 +3411,9 @@
 
       # Eliminate all temporary directories.
       for path in $notinst_path; do
-	lib_search_path=`$echo "$lib_search_path " | ${SED} -e "s% $path % %g"`
-	deplibs=`$echo "$deplibs " | ${SED} -e "s% -L$path % %g"`
-	dependency_libs=`$echo "$dependency_libs " | ${SED} -e "s% -L$path % %g"`
+	lib_search_path=`$echo " $lib_search_path " | ${SED} -e "s% $path % %g"`
+	deplibs=`$echo " $deplibs " | ${SED} -e "s% -L$path % %g"`
+	dependency_libs=`$echo " $dependency_libs " | ${SED} -e "s% -L$path % %g"`
       done
 
       if test -n "$xrpath"; then
